lib/src/svg/parsers.dart

```diff
@@ -153,21 +153,21 @@ Matrix4 _parseSvgRotate(String paramsStr, Matrix4 current) {
   final double a = radians(double.parse(params[0]));
 
   final Matrix4 rotate =
-      _affineMatrix(cos(a), sin(a), -sin(a), cos(a), 0.0, 0.0);
+      affineMatrix(cos(a), sin(a), -sin(a), cos(a), 0.0, 0.0);
 
   if (params.length > 1) {
     final double x = double.parse(params[1]);
     final double y = params.length == 3 ? double.parse(params[2]) : x;
-    return _affineMatrix(1.0, 0.0, 0.0, 1.0, x, y)
+    return affineMatrix(1.0, 0.0, 0.0, 1.0, x, y)
         .multiplied(current)
         .multiplied(rotate)
-        .multiplied(_affineMatrix(1.0, 0.0, 0.0, 1.0, -x, -y));
+        .multiplied(affineMatrix(1.0, 0.0, 0.0, 1.0, -x, -y));
   } else {
     return rotate.multiplied(current);
   }
 }
 
-Matrix4 _affineMatrix(
+Matrix4 affineMatrix(
```

Comment:
I think ultimately it would make the most sense to just move this function to one of the `path` libraries, but that's fine.  This shouldn't really hurt anything for now.